Changelog.md

```diff
@@ -1,5 +1,8 @@
 ### 0.7.3 (unreleased)
 
+Language Features:
+ * Support ``address(..).codehash`` to retrieve the code hash of an account.
+
```

Comment:
needs to rebase and the changelog moved to the new unreleased 0.7.4 (or breaking 0.8.0 if it is one)